x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form_fields.tsx

```diff
@@ -125,20 +121,18 @@ export const PipelineFormFields: React.FunctionComponent<Props> = ({
 
       {/* Pipeline Processors Editor */}
 
-      <PipelineProcessorsContextProvider
+      <ProcessorsEditorContextProvider
         onFlyoutOpen={onEditorFlyoutOpen}
         links={{ esDocsBasePath: services.documentation.getEsDocsBasePath() }}
+        api={services.api}
+        toasts={services.notifications.toasts}
```

Comment:
Great point. I agree. I'd like to get this PR in since it's needed for any subsequent PRs related to the pipeline simulation, but will definitely address in the next pass.